integration_tests/iceberg-sink/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM apache/airflow:2.6.2-python3.10 as airflow
+
+USER root
+
+# Install OpenJDK-11
+RUN apt-get update && \
+    apt-get install -y openjdk-11-jdk && \
+    apt-get install -y ant && \
+    apt-get clean;
+
+# Set JAVA_HOME
+ENV JAVA_HOME /usr/lib/jvm/java-11-openjdk-amd64/
+RUN export JAVA_HOME
+
+USER airflow
+
+RUN pip install --no-cache-dir "apache-airflow==${AIRFLOW_VERSION}" apache-airflow-providers-apache-spark==4.1.1 pyspark==3.3.1
+
+
+FROM bitnami/spark:3.3 as spark
+
+USER root
+
+RUN apt-get update && \
+    apt-get install -y curl && \
+    apt-get clean;
+
+RUN rm -r /opt/bitnami/spark/jars && \
+    curl https://ci-deps-dist.s3.amazonaws.com/spark-3.3.1-bin-hadoop3.tgz | \
```

Comment:
Any reason to replace the spark files with the one downloaded from CI?

Reply:
Because the Hadoop version in the image does not match the required version